tests/DocumentationExamplesTest.php

```diff
@@ -1408,6 +1410,57 @@ public function testTransactions_retry_example_3()
         }
     }
 
+    function testCausalConsistency()
+    {
+        $this->skipIfCausalConsistencyIsNotSupported();
+
+        // Prep
+        $client = new Client($this->getUri());
+        $test = $client->selectDatabase(
+            'test',
+            [ 'writeConcern' => new \MongoDB\Driver\WriteConcern(\MongoDB\Driver\WriteConcern::MAJORITY) ]
+        );
+        $items = $client->test->items;
+
+        $items->drop();
+        $items->insertOne(
+            [ 'sku' => '111', 'name' => 'Peanuts', 'start' => new \MongoDB\BSON\UTCDateTime() ]
+        );
+
+        // Start Causal Consistency Example 1
+        $s1 = $client->startSession([ 'causalConsistency' => true ]);
+
+        $items->updateOne(
+            [ 'sku' => '111', 'end' => [ '$exists' => false ] ],
+            [ '$currentDate' => [ 'end' => true ] ],
+            [ 'session' => $s1 ]
+        );
+        $items->insertOne(
+            [ 'sku' => '111-nuts', 'name' => 'Pecans', 'start' => new \MongoDB\BSON\UTCDateTime() ],
+            [ 'session' => $s1 ]
+        );
+        // End Causal Consistency Example 1
+
+        ob_start();
+
+        // Start Causal Consistency Example 2
+        $s2 = $client->startSession([ 'causalConsistency' => true ]);
+        $s2->advanceClusterTime($s1->getClusterTime());
+        $s2->advanceOperationTime($s1->getOperationTime());
+
+        $items = $client->selectDatabase(
+            'test',
+            [ 'readPreference' => new \MongoDB\Driver\ReadPreference(\MongoDB\Driver\ReadPreference::RP_SECONDARY) ]
+        )->items;
+
+        foreach ($items->find( [ 'end' => [ '$exists' => false ] ], [ 'session' => $s2 ] ) as $item) {
+            var_dump( $item );
```

Comment:
I realize this is your preferred coding style, but keep in mind that this may trigger warnings if we add code inspection to Travis down the line. I've no objections to spaces between array brackets and elements, as you've done above.

Side note: is this inconsistent with methods that take array arguments? For instance you have `startSession([ ...` above. Although you also have `$items->find( [ ...`, which aren't consistent with one another.

Reply:
I did remove the spaces inside the parenthesis here, and fixed the ``startSession`` case by having the arguments on their own line.